webapp/src/components/Modals/AssetFiltersModal/AssetFiltersModal.tsx

```diff
@@ -35,6 +45,7 @@ const AssetFiltersModal = (props: Props) => {
         />
       </Modal.Header>
       <Modal.Content>
+        {view === View.ACCOUNT ? <AssetTypeFilter onChange={handleAssetTypeChange} assetType={filters.assetType || assetType} /> : null}
```

Comment:
```suggestion
        {view === View.ACCOUNT ||  isWearableOrEmoteSection  ? <AssetTypeFilter onChange={handleAssetTypeChange} assetType={filters.assetType || assetType} /> : null}
```
I believe we need to show this also for the Collectible section (for wearables & emotes).
We could define a new variable like the one above:
```
const isWearableOrEmoteSection = useMemo(() => {
    return (
      view === View.MARKET &&
      (getCategoryFromSection(section) === NFTCategory.EMOTE ||
        getCategoryFromSection(section) === NFTCategory.WEARABLE)
    )
  }, [section, view])
```
For this, we would need to add `section` to the container.
Wdyt?

Reply:
Ignore my comment 😆 , we have the toggle outside the modal, so we're good :)  
![image](https://user-images.githubusercontent.com/8763687/212306348-ec8a7e41-2f2f-417e-972e-43723bb32cb0.png)